src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -24,6 +24,7 @@ public FindCommand(NameContainsKeywordsPredicate predicate) {
     @Override
     public CommandResult execute() {
         model.updateFilteredParcelList(predicate);
+        model.unselect();
```

Comment:
Why does find command have to unselect? oO

Reply:
The model will still have the flag "selected" if we don't switch it back.